### PR TITLE
Add @jameshowison as an admin on the UT hub

### PR DESCRIPTION
**Pull request by choldgraf**
This adds @jameshowison as an admin on the UT hub that @yuvipanda just deployed.

### CODE REVIEW

File: config/hubs/2i2c.cluster.yaml [hunk 1]
```diff
@@ -478,7 +478,9 @@ hubs:
             Authenticator:
               allowed_users: &utexas_demo_users
                 - <staff_github_ids>
+                - jameshowison
               admin_users: *utexas_demo_users
+
```
Comment (choldgraf) [inline config/hubs/2i2c.cluster.yaml:483]:
```suggestion
```

**Comment by colliand** [comment]
Ah, I misinterpreted. I thought I was to add James....

**Comment by choldgraf** [comment]
That's fine too - we've just often explicitly listed the community representatives in the config and let them add others. If you've already added them then this will not change the hub itself, only make it more explicit

**Review by yuvipanda — APPROVED** [review]
tyvm!

**Pull request merged**